Mathlib/Analysis/Normed/Ring/Lemmas.lean

```diff
@@ -161,12 +161,8 @@ instance (priority := 100) NonUnitalSeminormedRing.toContinuousMul [NonUnitalSem
         convert
           ((continuous_fst.tendsto x).norm.mul
                 ((continuous_snd.tendsto x).sub tendsto_const_nhds).norm).add
-            (((continuous_fst.tendsto x).sub tendsto_const_nhds).norm.mul _)
-        -- Porting note: `show` used to select a goal to work on
```

Comment:
`show` to select a goal actually does work, although the `·` linter complains. But I didn't see why we wouldn't just inline the argument.